supplier_app/migrations/0001_initial.py

```diff
@@ -66,11 +68,12 @@ class Migration(migrations.Migration):
             name='TaxPayerArgentina',
             fields=[
                 ('taxpayer_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='supplier_app.TaxPayer')),
-                ('cuit', models.CharField(max_length=200)),
-                ('comments', models.CharField(max_length=200)),
-                ('payment_type', models.CharField(max_length=200)),
+                ('cuit', models.CharField(max_length=20)),
+                ('payment_type', models.CharField(choices=[('BANK', 'Bank')], max_length=20)),
```

Comment:
I'm wondering if 'max_length' is still necessary or not

Reply:
Django ORM demands you to put a max_length